app/src/main/java/com/r5k/contacerveja/ui/drink/view/DrinkFragment.kt

```diff
@@ -78,4 +81,87 @@ class DrinkFragment : BaseFragment(),DrinkMVPView, View.OnClickListener {
             }
         }
     }
+
+    private fun showEditDialog(){
+
+        val builder = mContext?.let { AlertDialog.Builder(it) }
+
+        // https://stackoverflow.com/questions/10695103/creating-custom-alertdialog-what-is-the-root-view
+        // Seems ok to inflate view with null rootView
+        val view = layoutInflater.inflate(com.r5k.contacerveja.R.layout.dialog_new_drink, null)
```

Comment:
this package reference isn't necessary. 
```suggestion
        val view = layoutInflater.inflate(R.layout.dialog_new_drink, null)
```